soap/src/java/com/zimbra/soap/mail/message/FileSharedWithMeRequest.java

```diff
@@ -0,0 +1,160 @@
+/* 
+* ***** BEGIN LICENSE BLOCK *****
```

Comment:
fix indent

Reply:
done

Reply:
it has not been fixed.
```suggestion
 * ***** BEGIN LICENSE BLOCK *****
```

Reply:
done!